src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandName.java

```diff
@@ -0,0 +1,99 @@
+package io.stargate.sgv2.jsonapi.api.model.command;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * The canonical list of commands the API support, any command name not listed here is not a
+ * command.
+ */
+public enum CommandName {
```

Comment:
The enum name doesn't make sense, since it has more information than it's name, need to be renamed something like CommandInfo.

Reply:
this can be taken as separate PR as the name was an existing one. Created this [issue](https://github.com/stargate/data-api/issues/1701) for tracking.